hosted-tenant-base/pom.xml

```diff
@@ -36,7 +36,6 @@
         <target_jdk_version>17</target_jdk_version>
         <maven-compiler-plugin.version>3.10.1</maven-compiler-plugin.version>
         <maven-surefire-plugin.version>2.22.2</maven-surefire-plugin.version>
-        <junit.version>5.8.1</junit.version> <!-- NOTE: this must be in sync with junit version specified in 'tenant-cd-api' -->
```

Comment:
Suggest to instead rename to e.g. `vespa.junit.version` and add a comment that this must not be overridden by use apps.

Reply:
Sure.